src/lib.rs

```diff
@@ -17,6 +21,9 @@
 //! them with `Default::default()` and `None`.
 //!
 //! ```no_run
+//! # use wry::{Application, Attributes, Callback};
+//! # fn main() -> Result<(), wry::Error> {
+//! # let mut app = Application::new()?;
```

Comment:
There's a no run attribute. So you can remove these

Reply:
Inspite of having a no_run the doc-test failed to compile for me, hence added it.

Reply:
how about change `no_run` to other attributes like `ignore` or `text`?

Reply:
@wusyong `ignore` feels ok, but the docs could potentially become out-of-date. I'll remove this change from this PR for now, and maybe go over it in a separate one as it seems unrelated to #84